### PR TITLE
Cleanup: MMBN3 Typo for SloGauge

### PR DESCRIPTION
**Pull request by Hexadigital**
## What is this fixing or adding?
This fixes the Archipelago item name for SloGauge in Mega Man Battle Network 3 which was misspelled as SloGuage, causing the typo to appear in the Archipelago client and in games that display AP item names.

## How was this tested?
Not necessary due to the change only being swapping two characters in a single string.

## If this makes graphical changes, please attach screenshots.
N/A

### CODE REVIEW

**Review by nicholassaylor — CHANGES_REQUESTED** [review]
You missed a reference in `Items.py`
https://github.com/ArchipelagoMW/Archipelago/blob/06e65c1dc6ce4a1564d1b6924b83a0c9546011ec/worlds/mmbn3/Items.py#L174

**Comment by Hexadigital** [comment]
Sorry, I'm not sure what you're referencing there - would you like the variable name updated as well? I did not alter that in the file I edited.

**Comment by nicholassaylor** [comment]
> Sorry, I'm not sure what you're referencing there - would you like the variable name updated as well? I did not alter that in the file I edited.

I apologize for not being clear before. But yes, since the string is being corrected, the variable name in both `/Names/ItemName.py` and `Items.py` should probably be changed as well since they are both mispelled and the variables try to match the string closely. It's more a consistency thing rather than strictly correct or incorrect. Changing the names or not should not affect generation at all.

**Comment by Hexadigital** [comment]
Okay! Those have been adjusted as well.

**Review by nicholassaylor — APPROVED** [review]
Changes LGTM just waiting fit @digiholic to confirm the charges

**Review by Exempt-Medic — APPROVED** [review]
LGTM and covers every instance of `SloGuage`, just needs maintainer approval

**Comment by digiholic** [comment]
Ah, woops. I get so many messages about merges that I never saw this notification amongst them all. I need to clean up my email notifications...

**Review by digiholic — APPROVED** [review]

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**
